orchagent/orchdaemon.cpp

```diff
@@ -106,6 +120,7 @@ void OrchDaemon::start()
             for (Orch *o : m_orchList)
                 o->doTask();
 
+            flush();
```

Comment:
>flush(); [](start = 12, length = 8)

add comment here, something like below?

/* Let sairedis to flush all SAI function call to ASIC DB */